src/test/rgw/rgw_multi/tests_es.py

```diff
@@ -226,16 +227,16 @@ def create_obj(k, i):
                 do_check_mdsearch(target_conn.conn, bucket, src_keys , 'x-amz-meta-foo-date > ' + sval, lambda k: date_from_str(k.get_metadata('foo-date')) > val)
 
             # 'or' query
-            for i in xrange(len(src_keys) / 2):
```

Comment:
Not sure src_keys has a whatever big value... so no need in xrange and six.move.range here

Reply:
ditto.